cmd/query/app/http_handler.go

```diff
@@ -112,6 +114,9 @@ func NewAPIHandler(queryService *querysvc.QueryService, options ...HandlerOption
 	if aH.tracer == nil {
 		aH.tracer = opentracing.NoopTracer{}
 	}
+	if aH.tenancyConfig == nil {
```

Comment:
realistically, will we ever be in a situation where this is nil?  `cmd/query/app/server.go` does pass this option.

Reply:
I am happy to remove the defaulting; I will add an error return at the same time.